docs/source/using-executorch-windows.md

```diff
@@ -0,0 +1,149 @@
+# How to Build ExecuTorch for Windows
+
+This document outlines the current known working build instructions for building and validating ExecuTorch on a Windows machine.
+
+This demo uses the
+[MobileNet v2](https://pytorch.org/vision/main/models/mobilenetv2.html) model to
+process live camera images leveraging the
+[XNNPACK](https://github.com/google/XNNPACK) backend.
+
+Note that all commands should be executed on Windows powershell in administrator mode.
+
+## Pre-requisites
+
+### 1. Install Miniconda for Windows
+Install miniconda for Windows from the [official website](https://docs.conda.io/en/latest/miniconda.html).
+
+### 2. Install Git for Windows
+Install Git for Windows from the [official website](https://git-scm.com/download/win).
+
+### 3. Install ClangCL for Windows
+Install ClangCL for Windows from the [official website](https://learn.microsoft.com/en-us/cpp/build/clang-support-msbuild?view=msvc-170).
+
+
+## Create the Conda Environment
+To check if conda is detected by the powershell prompt, try `conda list` or `conda --version`
```

Comment:
Nit: Maybe reword as "To verify that Conda is available in the in the powershell environment, run  try `conda list` or `conda --version`. If Conda is not available, run `conda-hook.ps1` as follows: